tests/run/i5536/Macro_1.scala

```diff
@@ -6,7 +6,6 @@ object scalatest {
 
   def assertImpl(condition: Expr[Boolean])(implicit refl: Reflection): Expr[Unit] = {
```

Comment:
I imagine we still need to move it to `tests/run-macros/` (likewise with i5533b).